### PR TITLE
Scroll To Top button added on every page.

### PR DESCRIPTION
**Pull request by ChitvanRamani22**
# Description

I have added scroll to top button on each and every pahe that take user directly to top as shown in the video.

## Related Issues or Pull Requests

#145 

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [y ] Yes
- [ ] No

### Checklist

- [ y] I've read the contribution guidelines.
- [ y] I've checked the issue list before deciding what to submit.
- [ y] I've edited the `README.md` and link to my code.

I have added the video of one page this button is visible on every page.
https://user-images.githubusercontent.com/68965499/118270510-cc1a9b80-b4dd-11eb-8c9e-5b198bd8e35e.mp4
@Aasthajain123 @avinashkranjan please have a look at this pr.

### CODE REVIEW

**Comment by chetan-rns** [comment]
Thanks for the PR @ChitvanRamani22. Can you please update the commit message and add a description for the PR. Commit messages are helpful while debugging and should describe the change accurately.

**Comment by ChitvanRamani22** [comment]
@avinashkranjan done.

File: readme.md [hunk 1]
```diff
@@ -1,41 +1,109 @@
...
-12.	ca - number of major vessels (0-3) colored by flourosopy
-o	colored vessel means the doctor can see the blood passing through
-o	the more blood movement the better (no clots)
-13.	thal - thalium stress result
-o	1,3: normal
-o	6: fixed defect: used to be defect but ok now
-o	7: reversable defect: no proper blood movement when excercising
-14.	target - have disease or not (1=yes, 0=no) (= the predicted attribute)
-We have to collect this data of a patient. And we can use the model to predict whether the patient has the disease or not
+
```
Comment (chetan-rns) [inline readme.md:1]: Why are we updating the README?

**Review by chetan-rns — CHANGES_REQUESTED** [review]

**Comment by chetan-rns** [comment]
Please fix the merge conflict

File: readme.md (hunk 1 above)
Comment (ChitvanRamani22) [reply readme.md:1]: I havent done that i automatically gets updated everytime.

**Comment by ChitvanRamani22** [comment]
@avinashkranjan i have resolved the merge conflicts.

**Comment by avinashkranjan** [comment]
@ChitvanRamani22 Resolve the conflicts

**Comment by avinashkranjan** [comment]
@ChitvanRamani22 Kindly change the colour of background of the arrow to colour theme of the project..

**Comment by ChitvanRamani22** [comment]
@avinashkranjan shall i change it to blue?

**Comment by avinashkranjan** [comment]
> @avinashkranjan shall i change it to blue or black?

Use this color `#4a008f`

**Comment by ChitvanRamani22** [comment]
@avinashkranjan done!

**Review by avinashkranjan — APPROVED** [review]

**Pull request merged**